### ISSUE TITLE
CouchDB not installed on Windows

### ISSUE THREAD

**Issue by HTsybenko** [PASTA-ELN/pasta-eln#241]
When launching PASTA-ELN on Windows for the first time, CouchDB is not installed.

See the log file for the details.
[log.txt](https://github.com/PASTA-ELN/pasta-eln/files/14771001/log.txt)

**Comment by SteffenBrinckmann** [comment]
@HTsybenko Can you help me troubleshoot and try the following commands on the command line:
- "dotnet --list-sdks"
- "dotnet --list-runtimes"
- open "regedit" and go to branch "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full"; does it exist and what is the value of "Release" on the right side?
- Where did you find the log.txt file? I want to add this to the documentation

Notes:
- from the log-file it seems an old .net version is installed. I want to verify it.

**Comment by SteffenBrinckmann** [comment]
https://learn.microsoft.com/de-de/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed
says that the 0x8234d = 533325 version is larger than .net version 4.8.1
since the dotnet commands failed, version 5 of .net is not installed
likely, somewhere where in the middle.

**Comment by SteffenBrinckmann** [comment]
@HTsybenko can you try to follow this instruction, and see if it solves the issue
https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net80#install-with-windows-package-manager-winget

**Comment by HTsybenko** [comment]
@SteffenBrinckmann The Windows Package Manager was not installed on the PC. Therefore, I have installed .NET 8.0 via the download [link.](https://dotnet.microsoft.com/en-us/download/dotnet) The "dotnet --info" command shows that version 8.0.3 was installed.

After that, installing CouchDB during the initial PASTA-ELN configuration was not successful.
[log_CDB_1.txt](https://github.com/PASTA-ELN/pasta-eln/files/14849961/log_CDB_1.txt)

Manual installation of CouchDB using the wizard also does not work due to an unknown error.

**Comment by SteffenBrinckmann** [comment]
@HTsybenko can you try this:
https://learn.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows
afterwards reboot and retry, I know this is lower than 8; but according to 
https://github.com/apache/couchdb/issues/2554
it might help

**Comment by HTsybenko** [comment]
@SteffenBrinckmann With the 3.5, CouchDB was finally installed during the first time configuration. This solution has to be added to the documentation_ I guess.

**Comment by SteffenBrinckmann** [comment]
@HTsybenko Yes, it was added to documentation. Also, the code now checks if .NET 3.5 is installed and will stop and warn if not. TNX
